wire-library/wire-runtime-swift/build.gradle

```diff
@@ -19,8 +19,28 @@ dependencies {
   wire project(':wire-compiler')
 }
 
+task generateSwiftProtos(type: JavaExec) {
+  def swiftOut = 'src/main/swift/wellknowntypes'
+  def protoPath = 'src/main/proto'
```

Comment:
If we change this to like `../wire-schema/src/jvmMain/resources/google/protobuf/` then we can reuse the existing checked-in proto files. Then we can filter to only the messages we care about using something like:
```
--include=com.google.Duration,com.google.Instant
```
(messages are comma separated)

Reply:
Awesome. Are there benefits of reading a JAR over just having a hardcoded path to the resources directory?

Reply:
Should be the same. It just depends on which one you have available to you, and I think in this case we have a raw directory already.

Reply:
I would saying consuming via the jar is a little less bad since you are slightly less dependent on a bunch of hard-coded things that can change, but both still have tons of assumption in them. And both should fail in the same way if something moves so just go with the relative path I guess.